hoomd/md/pair/pair.py

```diff
@@ -293,6 +293,64 @@ def __init__(self, nlist, default_r_cut=None, default_r_on=0., mode='none'):
         super().__init__(nlist, default_r_cut, default_r_on, mode)
 
 
+class ExpandedGaussian(Pair):
+    r"""Expanded Gaussian pair force.
+
+    Args:
+        nlist (hoomd.md.nlist.NeighborList): Neighbor list.
+        default_r_cut (float): Default cutoff radius :math:`[\mathrm{length}]`.
+        default_r_on (float): Default turn-on radius :math:`[\mathrm{length}]`.
+        mode (str): Energy shifting/smoothing mode.
+
+    `ExpandedGaussian` computes the radially-shifted Gaussian pair force should
+    on every particle in the simulation state:
+
+    .. math::
+        U(r) = \varepsilon \exp \left( -\frac{1}{2}
+               \left( \frac{r-\delta}{\sigma} \right)^2 \right)
```

Comment:
```suggestion
               \left( \frac{r-\Delta}{\sigma} \right)^2 \right)
```
For consistency with `ExpandedLJ` and `ExpandedMie`.